src/core/state/currentSettings/featureFlags.ts

```diff
@@ -4,7 +4,6 @@ import { createStore } from '~/core/state/internal/createStore';
 
 export enum featureFlagTypes {
   full_watching_wallets = 'full_watching_wallets',
-  hw_wallets_enabled = 'hw_wallets_enabled',
```

Comment:
Did we want to deprecate this? IIRC, this was a fail safe incase transaction signing to and from hardware wallets was corrupted, leading to loss of funds, so we could at least temporarily disable the flow

Reply:
i honestly don't know. can make the other changes tho. what do u think